prac_06/languages.py

```diff
@@ -0,0 +1,14 @@
+from programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
```

Comment:
It seems you're trying to use a ProgrammingLanguage class from a module named programming_language. However, I don't have access to the content of this module, as it's a custom module that you've created or sourced from elsewhere.